apps/backend/src/modules/logger/logger.service.ts

```
@@ -0,0 +1,81 @@
+
+import { Environment } from "src/enums/environment.enum";
+import { LogMessage, LogType, RestLogStatus } from "src/enums/log.enum";
+import { PrismaService } from "src/modules/prisma/prisma.service";
+
+@Injectable({ scope: Scope.TRANSIENT })
+export class LoggerService extends ConsoleLogger {
+    private prisma = new PrismaService();
+
+    constructor() {
+        super();
+    }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)